testing/builtin_test.py

```diff
@@ -28,6 +28,19 @@
 assert abs(-3.4) == 3.4
 assert isinstance(abs(-3.4), float)
 
+assert abs(complex(0, 0)) == 0.0
```

Comment:
just removed 2 of assertions which are related to #262, will create a PR soon on #262 to solve it once forever